openassessment/xblock/test/test_submission.py

```diff
@@ -750,11 +751,28 @@ def test_get_team_submission_context(
     @scenario('data/submission_open.xml', user_id="Red Five")
     def test_get_team_submission_context__no_team(self, xblock):
         team_info = None
+        xblock.xmodule_runtime = Mock(
+            user_is_staff=False
+        )
         xblock.get_team_info = Mock(return_value=team_info)
         context = {}
         xblock.get_team_submission_context(context)
         self.assertEqual(context, {})
 
+    @scenario('data/submission_open.xml', user_id="Red Five")
+    def test_get_team_submission_context__staff_view(self, xblock):
```

Comment:
This test itself is not really explanatory, outside exceptions not being thrown. Would like to see a docstring or comment explaining what you're trying to assert.